server/lists/adapters/case-notes.js

```diff
@@ -202,7 +202,10 @@ const typeAdaptors = {
     OFFLINE_CASE_TRANSFER : ({ note }) => ({
         note: note,
         title: 'Offline Case Transfer'
-    })
+    }),
+    DATA_FIELD_UPDATED: ({ body }) => ({
```

Comment:
If this is a changed value, we should say that the field has changed from <value> to <new value>?

Reply:
Good question, this has come from the original ticket. I've asked Tom and we'll see what his opinion of changing it is